pjlib/build/pjlib.vcxproj.filters

```diff
@@ -1,449 +1,461 @@
+      <Filter>Source Files</Filter>
+    </ClCompile>
+    <ClCompile Include="..\src\pj\sock_select.c">
+      <Filter>Source Files</Filter>
+    </ClCompile>
+    <ClCompile Include="..\src\pj\ssl_sock_common.c">
+      <Filter>Source Files</Filter>
+    </ClCompile>
+    <ClCompile Include="..\src\pj\ssl_sock_dump.c">
+      <Filter>Source Files</Filter>
+    </ClCompile>
+    <ClCompile Include="..\src\pj\ssl_sock_ossl.c">
+      <Filter>Source Files</Filter>
+    </ClCompile>
+    <ClCompile Include="..\src\pj\string.c">
+      <Filter>Source Files</Filter>
+    </ClCompile>
+    <ClCompile Include="..\src\pj\timer.c">
+      <Filter>Source Files</Filter>
+    </ClCompile>
+    <ClCompile Include="..\src\pj\types.c">
+      <Filter>Source Files</Filter>
+    </ClCompile>
+    <ClCompile Include="..\src\pj\unicode_win32.c">
+      <Filter>Source Files</Filter>
+    </ClCompile>
+    <ClCompile Include="..\src\pj\unittest.c">
+      <Filter>Source Files</Filter>
+    </ClCompile>
+    <ClCompile Include="..\src\pj\addr_resolv_linux_kernel.c">
+      <Filter>Source Files\Other Targets</Filter>
+    </ClCompile>
+    <ClCompile Include="..\src\pj\guid_simple.c">
+      <Filter>Source Files\Other Targets</Filter>
+    </ClCompile>
+    <ClCompile Include="..\src\pj\ioqueue_dummy.c">
+      <Filter>Source Files\Other Targets</Filter>
+    </ClCompile>
+    <ClCompile Include="..\src\pj\ioqueue_epoll.c">
+      <Filter>Source Files\Other Targets</Filter>
+    </ClCompile>
+    <ClCompile Include="..\src\pj\log_writer_printk.c">
+      <Filter>Source Files\Other Targets</Filter>
+    </ClCompile>
+    <ClCompile Include="..\src\pj\os_core_unix.c">
+      <Filter>Source Files\Other Targets</Filter>
+    </ClCompile>
+    <ClCompile Include="..\src\pj\os_error_unix.c">
+      <Filter>Source Files\Other Targets</Filter>
+    </ClCompile>
+    <ClCompile Include="..\src\pj\os_timestamp_linux.c">
+      <Filter>Source Files\Other Targets</Filter>
+    </ClCompile>
+    <ClCompile Include="..\src\pj\pool_policy_kmalloc.c">
+      <Filter>Source Files\Other Targets</Filter>
+    </ClCompile>
+    <ClCompile Include="..\src\pj\symbols.c">
+      <Filter>Source Files\Other Targets</Filter>
+    </ClCompile>
+    <ClCompile Include="..\..\third_party\threademulation\src\ThreadEmulation.cpp">
+      <Filter>Source Files\winrt</Filter>
+    </ClCompile>
+    <ClCompile Include="..\src\pj\ip_helper_winphone8.c">
+      <Filter>Source Files</Filter>
+    </ClCompile>
+    <ClCompile Include="..\src\pj\ip_helper_generic.c">
+      <Filter>Source Files</Filter>
+    </ClCompile>
+    <ClCompile Include="..\src\pj\ip_helper_generic.c">
+      <Filter>Source Files</Filter>
+    </ClCompile>
+    <ClCompile Include="..\src\pj\ssl_sock_gtls.c">
+      <Filter>Source Files</Filter>
+    </ClCompile>
+    <ClCompile Include="..\src\pj\ssl_sock_imp_common.c">
+      <Filter>Source Files</Filter>
+    </ClCompile>
+    <ClCompile Include="..\src\pj\stack.c">

```

Comment:
It would be better if the patch doesn't modify the entire file, and only add the necessary ones.
Not sure how to do it though.

Same goes for the other `*.filters` file, and `pjlib_test.vcxproj` (strangely, `pjlib.vcxproj looks okay).